indexing-hadoop/src/main/java/io/druid/indexer/Utils.java

```diff
@@ -44,30 +40,19 @@
 {
   private static final ObjectMapper jsonMapper = new DefaultObjectMapper();
 
-  public static <K, V> Map<K, V> zipMap(Iterable<K> keys, Iterable<V> values)
```

Comment:
thanks, updated the description.

and, this code does not seem to get used anywhere. I can put it back if there is a reason to keep it around.
what do you think?